apps/full-site-editing/full-site-editing-plugin/full-site-editing/class-full-site-editing.php

```diff
@@ -478,25 +478,7 @@ public function merge_template_and_post( $post ) {
 			return;
 		}
 
-		$template_blocks = parse_blocks( $template_content );
-		$content_attrs   = $this->get_post_content_block_attrs( $template_blocks );
-
-		$wrapped_post_content = sprintf( '<!-- wp:a8c/post-content %s -->%s<!-- /wp:a8c/post-content -->', $content_attrs, $post->post_content );
-		$post->post_content   = str_replace( "<!-- wp:a8c/post-content $content_attrs /-->", $wrapped_post_content, $template_content );
-	}
-
-	/**
-	 * This will extract the attributes from the post content block
-	 * json encode them.
-	 *
-	 * @param array $blocks    An array of template blocks.
-	 */
-	private function get_post_content_block_attrs( $blocks ) {
-		foreach ( $blocks as $key => $value ) {
-			if ( 'a8c/post-content' === $value['blockName'] ) {
-				return count( $value['attrs'] ) > 0 ? wp_json_encode( $value['attrs'] ) : '';
-			}
-		}
+		$post->post_content = preg_replace( '@(<!-- wp:a8c/post-content)(.*?)(/-->)@', "$1$2-->$post->post_content<!-- /wp:a8c/post-content -->", $template_content );
```

Comment:
Optional: might be easier to read if we explicitly copy `<!-- wp:a8c/post-content` instead of `$1`. We'd also only need a capturing group for `.*?` in that case.